packages/nimble-components/src/drawer/styles.ts

```diff
@@ -153,33 +149,4 @@ export const styles = css`
         justify-content: flex-end;
         border-top: 2px solid rgba(${actionRgbPartialColor}, 0.1);
     }
-`.withBehaviors(
-    /*
-     * We cannot use the modalBackdropColor token directly because the backdrop
-     * element is not a descendant of the nimble-theme-provider element.
-     */
-    themeBehavior(
-        Theme.light,
-        css`
-            dialog::backdrop {
-                background: ${modalBackdropColorThemeLightStatic};
```

Comment:
@jattasNI I think we could potentially support the hard coded values as a fallback: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#defining_fallbacks_in_the_var_function

But I opted to just switch to the new pattern at the expense of chromatic diffs for the background for these stories (see PR description). I don't really feel strongly either way. I'd still like to clean out the fallback eventually so would keep the tech debt open if we used the fallback instead of what I proposed here.

Reply:
I couldn't find documentation on how often we expect Chromatic to upgrade so created https://github.com/chromaui/chromatic-cli/issues/1021

Reply:
Looks like they've historically updated just once a year in January or February. Maybe your issue will spur them to update more often, but I'm not too optimistic. I think I read somewhere once that the upgrades are expensive for them because they do a bunch of validation of what tests break across all of their customers suites.

Assuming they'll update early next year I suppose that's a fairly long time for our Chromatic baseline to be wrong. But also annoying to carry the tech debt of using a fallback for that long. I'm landing near the fence too but ultimately I'm ok with the current approach. 🚀

Reply:
Think it also helps that the Chromatic specific rendering difference is easy to evaluate / doesn't impact the primary content of the control. Seems unlikely to make future visual changes to those components difficult to evaluate. ⛴️ing